src/main/java/de/bonndan/nivio/input/compose2/ItemDescriptionFactoryCompose2.java

```diff
@@ -28,14 +32,22 @@
     }
 
     private final FileFetcher fileFetcher;
-    private final URL baseUrl;
 
-    public ItemDescriptionFactoryCompose2(FileFetcher fileFetcher, URL baseUrl) {
+    public ItemDescriptionFactoryCompose2(FileFetcher fileFetcher) {
         this.fileFetcher = fileFetcher;
-        this.baseUrl = baseUrl;
     }
 
-    public List<ItemDescription> getDescriptions(SourceReference reference) {
+    public static ItemDescriptionFactory forTesting() {
+        return new ItemDescriptionFactoryCompose2(new FileFetcher(new HttpService()));
+    }
+
+
+    @Override
+    public List<String> getFormats() {
+        return Arrays.asList("docker-compose-v2");
```

Comment:
```suggestion
        return Collections.singletonList("docker-compose-v2");
```

Reply:
Sure we will have only one format name here?